storage/CSBTreeIndexSubBlock.cpp

```diff
@@ -353,6 +355,12 @@ std::size_t CSBTreeIndexSubBlock::EstimateBytesPerTuple(
   return (5 * key_length) >> 1;
 }
 
+std::size_t CSBTreeIndexSubBlock::EstimateBytesPerBlock(
+    const CatalogRelationSchema &relation,
+    const IndexSubBlockDescription &description) {
+  return 0;
```

Comment:
@cramja Sure, it makes sense to add a boolean flag indicating this. But I am still struggling to find a proper place to put this variable. IndexSubBlock.hpp seems to only define virtual functions. and EstimateBytesPerTuple() and EstimateBytesPerBlock() are not even defined in the base class itself. Maybe we can put it in each derived class file.